src/components/generic-card/index.tsx

```diff
@@ -1,25 +1,31 @@
 import { SpacingProp } from "@/types/style";
 import { H2, P } from "../text";
 import s from "./GenericCard.module.css";
+import Image, { ImageProps } from "next/image";
 
 interface GenericCardProps {
   title: string;
+  image?: ImageProps;
   description: string;
   padding?: SpacingProp;
   children?: React.ReactNode;
 }
 
 export default function GenericCard({
   title,
+  image,
   padding = "56px",
   description,
   children,
 }: GenericCardProps) {
   return (
-    <div className={s.genericCard} style={{ padding }}>
-      <H2 className={s.title}>{title}</H2>
-      <P className={s.description}>{description}</P>
-      {children ? <div className={s.children}>{children}</div> : null}
+    <div className={s.genericCard}>
+      {image ? <Image {...image} /> : null}
```

Comment:
The adjustments that we have to make here are telling me that this generic card is probably a premature abstraction.

I recommend just building out a distinct component for your card needs for right now.

Once we land on some consistency in our cards, then we should make this abstraction.

Reply:
Yeah makes sense, will revert the changes later tonight 😅

Reply:
It was a worthy endeavor to try though, so all good!